backend/src/agents/__init__.py

```diff
@@ -28,7 +29,9 @@ def agent_details(agent) -> dict:
 
 def get_available_agents() -> List[Agent]:
     return [DatastoreAgent(config.datastore_agent_llm, config.datastore_agent_model),
-            MathsAgent(config.maths_agent_llm, config.maths_agent_model)]
+            WebAgent(config.web_agent_llm, config.web_agent_model),
+            # MathsAgent(config.maths_agent_llm, config.maths_agent_model)

```

Comment:
Was this meant to be commented out?

Reply:
Yes, we decided we don't need Maths agent at this moment, so I confirmed with @hsauve-scottlogic before commenting it out.

Reply:
Yes it's causing issues with queries that are thought to be related to maths rather than database. We could remove line 33 completely, the MathsAgent is still in /agents, it just won't be used for now, see what you think.

Reply:
Have removed the line completely.